polymer.json

```diff
@@ -7,5 +7,14 @@
     "src/shop-cart.html",
     "src/shop-checkout.html",
     "src/lazy-resources.html"
+  ],
+  "sourceGlobs": [
+   "src/**/*",
+   "images/**/*",
+   "bower.json"
+  ],
+  "includeDependencies": [
+    "manifest.json",
+    "bower_components/webcomponentsjs/webcomponents-lite.min.js"
```

Comment:
@FredKSchott Thanks man!
@robdodson Do you mind updating to add `"data/**/*"` in the PR?